src/modeler/AutoRest.Swagger/Validation/OperationIdNounConflictingModelNames.cs

```diff
@@ -26,6 +26,18 @@ public class OperationIdNounConflictingModelNames : TypedRule<string>
         public override ValidationCategory ValidationCategory => ValidationCategory.SDKViolation;
 
         /// <summary>
+        /// What kind of open api document type this rule should be applied to
+        /// </summary>
+        public override ServiceDefinitionDocumentType ServiceDefinitionDocumentType => ServiceDefinitionDocumentType.ARM;
```

Comment:
is this ARM only? I had thought the issue come in due to the way AutoRest picks up these names. I know there's a recent thread on this rule in terms of being a real conflict or not, so we should figure that out.